### DISCUSSION TITLE
Will this driver ever be GA?

### DISCUSSION THREAD

**Discussion by nfdavenport** [awslabs/aws-postgresql-jdbc#66] [General]
The 0.1.0 release is coming up on a year old.  It seems most development has been going on in the MySQL driver. We found this driver 6 months ago and have been hoping it would help solve the sporadic production issues we have during failover caused by DNS race conditions.

**Comment by hsuamz** [comment]
For visibility, we are recommending everyone to move to our new repository here: https://github.com/awslabs/aws-advanced-jdbc-wrapper.

We have redesigned the driver as a JDBC Wrapper which has been tested to work with the PGJDBC driver. Please give it a look and we will be continuing discussion there.

Thanks.
